### PR TITLE
Added Log file Tip to cmpivot.md

### LINKED ISSUE #742: cmpivot related log files
The logs for troubleshooting are:
**Server side**:
- Smsprov.log
- bgbserver.log
- Statesys.log

**Client side**:
- CcmNotificationAgent.log
- Scripts.log
- StateMessage.log

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f0c8158c-1c8e-1567-d96d-222f5b4aec21
* Version Independent ID: f6674943-3564-2250-2334-b88c42f3a8a6
* Content: [CMPivot for real-time data - Configuration Manager](https://docs.microsoft.com/en-us/sccm/core/servers/manage/cmpivot#feedback)
* Content Source: [sccm/core/servers/manage/cmpivot.md](https://github.com/Microsoft/SCCMdocs/blob/master/sccm/core/servers/manage/cmpivot.md)
* Product: **configuration-manager**
* GitHub Login: @aczechowski
* Microsoft Alias: **aaroncz**

### PR DESCRIPTION
**Pull request by onpremcloudguy**
Fixes #742

### CODE REVIEW

**Comment by PRMerger9** [comment]
@onpremcloudguy : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

File: sccm/core/servers/manage/cmpivot.md [hunk 1]
```diff
@@ -224,6 +224,20 @@ The queries and the results are all just text. The entities **InstallSoftware**
 
 A query times out after one hour. For example, a collection has 500 devices, and 450 of the clients are currently online. Those active devices receive the query and return the results almost immediately. If you leave the CMPivot window open, as the other 50 clients come online, they also receive the query, and return results. 
 
+>[!TIP]
+> CMPivot interations are logged to the following log files:
+
```
Comment (aczechowski) [inline sccm/core/servers/manage/cmpivot.md]: Please add a `>` here to do a contiguous quoted section.

File: sccm/core/servers/manage/cmpivot.md [hunk 2]
```diff
@@ -224,6 +224,20 @@ The queries and the results are all just text. The entities **InstallSoftware**
...
+> Server Side:
```
Comment (aczechowski) [inline sccm/core/servers/manage/cmpivot.md]: Please either change this to just "Server" or "Server-side". And maybe bold it (surround with `**` tags) too.

File: sccm/core/servers/manage/cmpivot.md [hunk 3]
```diff
@@ -224,6 +224,20 @@ The queries and the results are all just text. The entities **InstallSoftware**
...
+> - SmsProv.log
+> - bgbServer.log
+> - StateSys.log
+
```
Comment (aczechowski) [inline sccm/core/servers/manage/cmpivot.md]: Please add a `>` here to do a contiguous quoted section.

File: sccm/core/servers/manage/cmpivot.md [hunk 4]
```diff
@@ -224,6 +224,20 @@ The queries and the results are all just text. The entities **InstallSoftware**
...
+> Client Side:
```
Comment (aczechowski) [inline sccm/core/servers/manage/cmpivot.md]: Please either change this to just "Client" or "Client-side" (to match the similar header above). Keep formatting the same as well (bold).

File: sccm/core/servers/manage/cmpivot.md [hunk 5]
```diff
@@ -224,6 +224,20 @@ The queries and the results are all just text. The entities **InstallSoftware**
...
+> - CcmNotificationAgent.log
+> - Scripts.log
+> - StateMessage.log
+
```
Comment (aczechowski) [inline sccm/core/servers/manage/cmpivot.md]: Please add a `>` here to do a contiguous quoted section.

File: sccm/core/servers/manage/cmpivot.md [hunk 6]
```diff
@@ -224,6 +224,20 @@ The queries and the results are all just text. The entities **InstallSoftware**
...
+> [For more Information on Configuration Manger log files](/sccm/core/plan-design/hierarchy/log-files)
```
Comment (aczechowski) [inline sccm/core/servers/manage/cmpivot.md]: Please revise this to: `For more information, see [Log files](/sccm/core/plan-design/hierarchy/log-files).`

**Comment by PRMerger10** [comment]
@onpremcloudguy : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by aczechowski** [comment]
Thanks for the updates! I either missed it the first time or lost it in the update, I think let's bring the last line into the tip as well. So add `>` to the last two lines (the empty space and the "for more info..."). 

And while you're making one more edit, drop the case on "side". In other words, "Server-side" and "Client-side". [Reference](https://docs.microsoft.com/en-us/style-guide/top-10-tips-style-voice#when-in-doubt-dont-capitalize)

**Comment by PRMerger14** [comment]
@onpremcloudguy : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by aczechowski** [comment]
Looks great, thanks @onpremcloudguy for the quick edits!
#sign-off

**Comment by aczechowski** [comment]
I forgot to mention: *Once merged*, this should be live with the next daily publishing process. Right now that *should* be tomorrow (Friday, 10 August) around 11:00 Pacific time.

**Pull request merged**